test/SharedEventManagerTest.php

```diff
@@ -27,7 +27,7 @@ public function setUp()
 
     public function getListeners(SharedEventManager $manager, array $identifiers, $event, $priority = 1)
     {
-        $priority = (int) $priority . '.0';
+        $priority = (int) $priority;
```

Comment:
Note to self: this will affect every single component we've upgraded that implements a "getListeners" method of some sort for introspecting attached listeners.